### ISSUE TITLE
Requested URL from WMS (GetFeatureInfo) not working

### ISSUE THREAD

**Issue by andrearivera8** [Mapsui/Mapsui#408]
Hello.
I am using a function for a GetFeatureInfo request (https://github.com/Mapsui/Mapsui/blob/master/Mapsui.Desktop/Wms/GetFeatureInfo.cs) to received information from a Geoserver WMS. 

Once the function receives the url requested, it does not parser to text/xml because the url obtained does not work. I have tried the url in a web browser and I get this error: 
![image](https://user-images.githubusercontent.com/38493552/39814798-04be5bf4-5396-11e8-8bfa-fc179c471da3.png)

Thanks!

**Comment by pauldendulk** [comment]
Is there a way I could reproduce this? Is this a public service?

**Comment by andrearivera8** [comment]
No, is not a public service, but my Geoserver is inside a Tomcat. Is it not enough with the IP adress?

**Comment by pauldendulk** [comment]
So the url return a 404? The url used for the feature request comes from the wms capabilities xml. At startup the layer fetches that this. What you could do is to take a look in the wms capabilities xml and see what feature info url is used there and why it might now work. 

Perhaps you can use qgis. Does the feature info work there? What url is it using to fetch the feature info? How is it different from what Mapsui is using?

**Comment by charmosz** [comment]
Hi. Please could you share how you get the feature info ? Thanks !
